### PR TITLE
Update setup to tdlib 1.7.0

### LINKED ISSUE #325: Upgrade to tdlib-1.7.0
Going to version 1.7.0 of tdlib allows us to order chats correctly.
“The user wants a fast development process. For that, documented apis are needed”

### LINKED ISSUE #326: CI Pipeline is broken
**Describe the bug**
CI fails on most platforms

**To Reproduce**
1. run CI on GitHub

**Expected behavior**
All tests pass

**Additional context**
Not relevant to user directly, might impact testing and therefore quality.

### PR DESCRIPTION
**Pull request by rsommerfeld**
Fixes CI Pipeline by using new artifacts and updating tests regarding to changed tdlib testing and interface.

closes #325
closes #326 

ToDo:
- [x] refactor attribute "can_send_messages" of TCCChat
- [x] refactor changes regarding to chat order
- [x] adapt wait times of testing to more modest approximations

### CODE REVIEW

File: packages/TelegramClient-Tests.package/TCTAuthTests.class/instance/defaultTimeout.st [hunk 1]
```diff
@@ -0,0 +1,4 @@
+accessing
+defaultTimeout
+
+	^ 10
```
Comment (jb3rndt) [inline packages/TelegramClient-Tests.package/TCTAuthTests.class/instance/defaultTimeout.st:4]: Is this used anywhere? If not we could probably remove it.
Or is it a method the testing framework implicitly uses?

File: packages/TelegramClient-Tests.package/TCTChatsTests.class/instance/testPositionInsert.st [hunk 2]
```diff
@@ -0,0 +1,9 @@
+testing
+testPositionInsert
+
+	self chats add: (TCCChat new
+		id: 0;
+		position: 0;
+		positionKnown: true).
+
+	self assert: self chats first id equals: 0.
```
Comment (jb3rndt) [inline packages/TelegramClient-Tests.package/TCTChatsTests.class/instance/testPositionInsert.st:9]: I think we want the assert to have the constant as the first argument.

File: packages/TelegramClient-Tests.package/TCTLoggedInTests.class/instance/defaultTimeout.st [hunk 3]
```diff
@@ -0,0 +1,4 @@
+accessing
+defaultTimeout
+
+	^ 40
```
Comment (jb3rndt) [inline packages/TelegramClient-Tests.package/TCTLoggedInTests.class/instance/defaultTimeout.st:4]: Same as above?

**Review by jb3rndt — CHANGES_REQUESTED** [review]

File: packages/TelegramClient-Tests.package/TCTChatsTests.class/instance/testPositionInsert.st (hunk 2 above)
Comment (tom-richter) [reply packages/TelegramClient-Tests.package/TCTChatsTests.class/instance/testPositionInsert.st:9]: Resolved

File: packages/TelegramClient-Tests.package/TCTLoggedInTests.class/instance/defaultTimeout.st (hunk 3 above)
Comment (tom-richter) [reply packages/TelegramClient-Tests.package/TCTLoggedInTests.class/instance/defaultTimeout.st:4]: same answer as above

File: packages/TelegramClient-Tests.package/TCTAuthTests.class/instance/defaultTimeout.st (hunk 1 above)
Comment (rsommerfeld) [reply packages/TelegramClient-Tests.package/TCTAuthTests.class/instance/defaultTimeout.st:4]: It is inherited from the Testing Framework and used to determine the timeout for the test. You can also use the pragmas (<timeout>) or send a message in setUp ("timeout:"). The former is weird and the latter (which was previously used) did not work on our machines. I suppose it must have worked at some point, but using the method seemed like the better solution to us 😄 

PS: you can check for inheritance in the browser

Comment (jb3rndt) [reply packages/TelegramClient-Tests.package/TCTAuthTests.class/instance/defaultTimeout.st:4]: Yeah I only looked into the changes on this PR and did not start my image 😅
But yeah, nice 👍

**Review by jb3rndt — APPROVED** [review]

**Pull request merged**